auth/realm/jdbc/src/main/java/org/wildfly/security/auth/realm/jdbc/mapper/PasswordKeyMapper.java

```diff
@@ -66,11 +67,6 @@
     private final Encoding hashEncoding;
     private final Encoding saltEncoding;
 
-    public enum Encoding {
```

Comment:
Just FYI for other reviewers, I've checked that `PasswordKeyMapper` isn't public API.